packages/frontend/core/src/modules/explorer/views/nodes/collection/index.tsx

```diff
@@ -251,7 +251,7 @@ const ExplorerCollectionNodeChildren = ({
       [docsService]
     )
   );
-  const favourites = useLiveData(favoriteItemsAdapter.favorites$);
+  const favourites = useLiveData(compatibleFavoriteItemsAdapter.favorites$);
```

Comment:
favorites

Reply:
![CleanShot 2024-07-26 at 15 49 38](https://github.com/user-attachments/assets/176e061c-2cf5-4255-94ad-60701304c5d2)
too many, i will do that in other PR